docs/man/encrypt/_index.md

```diff
@@ -24,6 +24,8 @@ command:
     - name: kas-url-path
       description: URL path to the KAS service at the platform endpoint domain. Leading slash is required if needed.
       default: /kas
+    - name: with-assertions
```

Comment:
Can you please add using the flag to the docs for the help manual output below to show what the assertions JSON structure should look like?

Reply:
sure